vortexasdk/retry_session.py

```diff
@@ -31,11 +31,14 @@ def _requests_retry_session(
     return session
 
 
+_headers = {"Content-Type": "application/json"}
```

Comment:
`nit`: If this is a global constant, convention is to set it as `_HEADERS` at the top of the file